### ISSUE TITLE
improve BNN results

### ISSUE THREAD

**Issue by miro-4231** [BIGmama-technology/Hiring-AI-engineer#13]
السلام عليكم
can we use a dropout and/or a batch normalization layer with a simple MLP instead of a BNN?? according to these papers 
 [dropout](https://arxiv.org/abs/1506.02142)  , [batch_normalization](https://arxiv.org/abs/1802.06455) it's an approximation
thank you in advance

**Comment by 1hachem** [comment]
Oh its a nice idea ! we can add it to the tasks, however you have to make sure to visualize the uncertainty of the model.

**Comment by miro-4231** [comment]
OK!!

**Comment by 1hachem** [comment]
lessss goooooo 🚀

**Comment by 1hachem** [comment]
going to close this !
